### ISSUE TITLE
Docker image broken (missing dependency)

### ISSUE THREAD

**Issue by TimWolla** [letsencrypt/letsencrypt#1678]
The most recent version of the Docker image is broken. When I try to run it the following error message appears:

```
$ docker run -it --rm quay.io/letsencrypt/letsencrypt:latest help
Traceback (most recent call last):
  File "/opt/letsencrypt/venv/bin/letsencrypt", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/opt/letsencrypt/venv/local/lib/python2.7/site-packages/pkg_resources.py", line 2720, in <module>
    parse_requirements(__requires__), Environment()
  File "/opt/letsencrypt/venv/local/lib/python2.7/site-packages/pkg_resources.py", line 588, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: acme==0.1.1
```

**Comment by TimWolla** [comment]
Looking further into it: I suspect it being related to the differing version numbers in these files:

https://github.com/letsencrypt/letsencrypt/blob/9c687a2a7db3b7b3a00525fc9b0d86ca594b0ef5/acme/setup.py#L7
https://github.com/letsencrypt/letsencrypt/blob/9c687a2a7db3b7b3a00525fc9b0d86ca594b0ef5/letsencrypt/__init__.py#L4

**Comment by TimWolla** [comment]
Fixed as of c45e394824a9e05aa169bc9ca67dcccf95041c2a
